_utils/firefox_profile.py

```diff
@@ -0,0 +1,93 @@
+"""
+This module uses Fabric API to generate a Firefox Profile by concatenating the
+following preferences files:
+- ./_utils/prefs.ini
+- ./<application>/prefs.ini
+- ./<application>/<test_type>/prefs.ini
+
+The profile is then created using the specified name and saved to the ../_temp/
+directory.
+"""
+
+try:
+    import configparser  # Python 3
+except:
+    import ConfigParser as configparser  # Python 2
+
+import os
+import shutil
+
+import configargparse
+from fabric.api import local  # It looks like Fabric may only support Python 2.
```

Comment:
@rpappalax This was weird. It seems like Fabric isn't Python 3 compatible yet. But this worked locally in Python 2.